apps/web/src/pages/integrations/useProviders.ts

```diff
@@ -41,7 +41,10 @@ function initializeProviders(integrations: IntegrationEntity[]): IIntegratedProv
       if (integration?.credentials && Object.keys(clonedCredentials).length !== 0) {
         clonedCredentials.forEach((credential) => {
           // eslint-disable-next-line no-param-reassign
-          credential.value = integration.credentials[credential.key]?.toString();
+          credential.value =
+            credential.type === 'boolean' || credential.type === 'switch'
+              ? integration.credentials[credential.key]
+              : integration.credentials[credential.key]?.toString();
```

Comment:
found this issue in the way, we stringify boolean params which is wrong, switch component's in the form will think that "false" is truthy.
it should be in prod as well IMO but i could not reproduce it, could be related to the FF's

Reply:
yeah this one have been hitting me before a tricky one and does not seems to happen in all places.

Reply:
Very strange it should happen where we initializeProvidersByIntegration which should happen when isMultiProviderConfigurationEnabled.

Reply:
`initializeProviders` is called when the `isMultiProviderConfigurationEnabled: false`, we have a similar one `initializeProvidersByIntegration` but it's fixed there

Reply:
> `initializeProviders` is called when the `isMultiProviderConfigurationEnabled: false`, we have a similar one `initializeProvidersByIntegration` but it's fixed there

totally right i mismatched the function names 😔